scripts/createTransaction.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config();
+const Transaction = require("../classes/transaction");
+
+function newTransaction(to, from, nonce, value, data, gasLimit, gasPrice) {
```

Comment:
I dont see any invocation point of this method. Do check if this has to be used?